<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseConfig</class>
 <widget class="QDialog" name="DatabaseConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konfigurasi Database</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../glypticon.qrc">
    <normaloff>:/glypticon/glyphicons_136_cogwheel.png</normaloff>:/glypticon/glyphicons_136_cogwheel.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="lHost">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lUser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>119</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>root</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lPass">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>148</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>root</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>89</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>119</y>
     <width>54</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>149</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>180</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>89</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Port </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox">
      <property name="toolTip">
       <string>Port: Biarkan Saja Jika Menggunakan Default MySql</string>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>229</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pSimpan">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Konek</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pTest">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Batal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lDatabase">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>179</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>database</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
color: green;</string>
   </property>
   <property name="text">
    <string>Setting Koneksi Database</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.492, y1:0, x2:0.492, y2:1, stop:0 rgba(85, 255, 0, 200), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pengaturan Koneksi Database MySQL</string>
   </property>
  </widget>
  <zorder>layoutWidget1</zorder>
  <zorder>lHost</zorder>
  <zorder>lUser</zorder>
  <zorder>lPass</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_5</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>lDatabase</zorder>
  <zorder>label_7</zorder>
  <zorder>label_6</zorder>
  <zorder>label_8</zorder>
 </widget>
 <tabstops>
  <tabstop>lHost</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>lUser</tabstop>
  <tabstop>lPass</tabstop>
  <tabstop>lDatabase</tabstop>
  <tabstop>pSimpan</tabstop>
  <tabstop>pTest</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="../../glypticon.qrc"/>
 </resources>
 <connections/>
</ui>
